ASTProcessor/src/main/java/edu/usb/argos/ASTProcessor/lexer/errorHandler/ArgosErrorListener.java

```diff
@@ -0,0 +1,38 @@
+package edu.usb.argos.ASTProcessor.lexer.errorHandler;
+
+import edu.usb.argos.ASTProcessor.lexer.errorHandler.exceptions.AntlrException;
+import org.antlr.v4.runtime.*;
+import org.antlr.v4.runtime.atn.ATNConfigSet;
+import org.antlr.v4.runtime.dfa.DFA;
+
+import java.util.BitSet;
+
+public class ArgosErrorListener implements ANTLRErrorListener {
```

Comment:
Use @Slf4 for logger, also request by trainer. https://www.slf4j.org/

![image](https://github.com/user-attachments/assets/60f6344d-f424-479a-9349-db8136bd70f1)

Reply:
Done ✅